### ISSUE TITLE
Telling Monika (politely) you're not comfortable with her wearing certain outfits.

### ISSUE THREAD

**Issue by Retrolovania** [Monika-After-Story/MonikaModDev#7738]
Given the more "adult" nature of some of the unlockable outfits Monika can don when higher affection levels are achieved, some individuals who aren't comfortable with such things may be put off with her expecting the player in question to find it appealing. Since Sensitive Mode is no longer around to automatically prevent such a thing from appearing game-wise, I think a more down-to-earth solution to this would be to have a selection prompt when Monika asks if you like her outfit when it first appears. Selecting `Yes` would prompt her to remember to keep it on standby, whilst `No` would inform her to not wear it afterwards.

**Comment by Booplicate** [comment]
Right now she wears what you asked her to.

**Comment by Retrolovania** [comment]
While that is true, that doesn't really account for initial reveal.

**Comment by Booplicate** [comment]
You suggest to prevent all future unlocks if the player's said they don't like such outfits?

**Comment by Retrolovania** [comment]
...no?
I'm only suggesting that if the player doesn't like a new outfit, then they can respond accordingly and have it never appear again. Wouldn't apply to *every* single next unlock.

**Comment by lunulae** [comment]
I could see maybe a more generally accessible toggle for showRisque, for those who are uncomfortable with suggestive content past the age limit. I've seen some discussion of letting her know you're asexual or you're just not ready for that step yet, with the option to give that consent later. 

It might still need some consideration since it would still mean folks would have to wait for the holiday to roll around again to get the unlock if they changed their mind, but if they've used the option it's generally understood they'll agree to that.

**Comment by lunulae** [comment]
To my memory however, the lingerie outfits don't prompt on their own after their first showing even on the holiday. This would purely function for first unlock and if they would show up in the outfit selector.

**Comment by ibm2431** [comment]
Any reply from the player needs to be carefully crafted.

> "I put a lot of effort into looking nice for you. Do you like it?"

Who'd be okay with tossing out their partner's feelings and efforts with just a simple "No"? It'd be like her own knitted sweater curse.

A better approach would be Monika bringing up the subject prior to her attempt. She's pretty good about asking for consent.

She doesn't need to ask _right before_, ruin any surprise, and/or accidentally apply pressure on the player:
> "It's `[special day]`! Actually, [player]... I... have this outfit I want to wear for you. Is that okay?"

But she can feel out the player in the days leading up to it, even indirectly:
> "Hey... this might sound weird... and let me know if you're uncomfortable with this... but how do you feel about `type of outfit`?"

**Comment by TheGuy44** [comment]
so you don't like what outfits she wears? my god what was the world come to unless it has derogatory things and/or words on it then guess it's understandable.

**Comment by Retrolovania** [comment]
...I'd like to think some decency still exists.

**Comment by CrewieAustin** [comment]
> so you don't like what outfits she wears? my god what was the world come to unless it has derogatory things and/or words on it then guess it's understandable.

it's not that retro don't like it, just a suggestion retro would like to have in the game. and yea, some people do have certain opinions about some outfits. like how hoodies being bad while the others think it's comfy and good.

**Comment by TheGuy44** [comment]
fair enough. but at the same time people should grow a pair. plus not like she can wear most outfits by herself except as far as we for the valentine's day dress, black date dress, santa lingerie, various halloween costumes over the last three years, and the new years' eve costume. don't know those outfits can make people uncomfortable but oh well.

**Comment by ThePotatoGuy** [comment]
I like @ibm2431 's idea. And we can setup a pool topic to ask her to wear something more suggestive if you originally declined.
